src/compile.js

```diff
@@ -525,6 +525,15 @@ Compiler.prototype.ccall = function (e) {
     }
     else {
         out ("$ret;"); // This forces a failure if $ret isn't defined
+        if (Sk.python3 && e.func.id && e.func.id.v === "super" && args.length == 0) {
```

Comment:
Fair enough.  Can we add a clear comment here, then, about the limitation that it will not work if the Python code violates the ```self``` convention?